events/galaxy-academy-2024.md

```diff
@@ -1,54 +1,124 @@
+      name: Galaxy AU
+
+program:
+  - section: "Monday: Galaxy introduction"
     description: |
-      Today will be all about getting to know Galaxy and explore its potentials. Additionally you can learn about the foundation of data analysis. 
+      Kick off the week with a hands-on introduction to Galaxy, covering everything from basic navigation and data manipulation to reproducing published analyses, quality control, and mapping sequences to a reference genome. Whether you're new to Galaxy or looking to strengthen your skills, today's sessions will equip you with the foundational knowledge needed for more advanced topics.
     tutorials:
-      - name: galaxy-intro-short
+      - type: custom
+        name: "Start to get to know Galaxy "
+      - name: galaxy-intro-101-everyone
+        topic: introduction
+      - name: data-manipulation-olympics
         topic: introduction
-      - name: galaxy-intro-101
+      - name: galaxy-reproduce
         topic: introduction
+      - name: options-for-using-galaxy
+        topic: introduction
+      - type: custom
+        name: "Fundamentals of Sequences analysis "
       - name: quality-control
         topic: sequence-analysis
       - name: mapping
         topic: sequence-analysis
  
-  - section: "Tuesday: Transcriptomics"
+  - section: "Tuesday to Thursday: Pick a track"
     description: Today you will learn all about transcriptomics
-    tutorials:
-      - name: ref-based
-        topic: transcriptomics
-
-  - section: "Wednesday: Proteomics"
-    description: 
-    tutorials:
-      - name: introduction
-        topic: proteomics
-
-  - section: "Thursday: Microbiome"
-    description: 
-    tutorials:
-      - name: introduction
-        topic: microbiome
+    tracks:  # instead of tutorials, you can also define tracks, this will create a button per tracks that will lead to a different page
+      - title: Proteomics
+        link: events/tracks/gta2024-proteomics.md
+      - title: Assembly
+        link: events/tracks/gta2024-assembly.md
+      - title: Transcriptomics
+        link: events/tracks/gta2024-transcriptomics.md
+      - title: Single Cell
+        link: events/tracks/gta2024-single-cell.md
+      - title: Microbiome
+        link: events/tracks/gta2024-micorbiome.md
+      - title: Bacterial Genomics
+        link: events/tracks/gta2024-bacterial-genomeics.md
+      - title: BY-COVID
+        link: events/tracks/gta2024-bycovid.md
+      - title: Machine Learning
+        link: events/tracks/gta2024-ml.md
 
   - section: "Friday: Grab bag"
     description: | 
-      Here we have some additional topics. Please feel free to pick and choose the tutorials that are interesting for you.
-    tutorials:
-      - name: troubleshooting
-        topic: admin
-
+      Can't get enough? Then please pick any of the tutorials of the GTN. Please be aware that we can only give consider tutorials of the first day or any that is part of a Track of the Galaxy Training Academy 2024.
```

Comment:
I am not sure what you mean here. Something like : Please be aware that only trainings that are part of the introduction day or a learning track have been tested on all instances for the event. The trainers present on Slack will do their best to help you if you have a problem and answer questions, but they may not be expert in the topic you selected.

Reply:
@Delphine-L  Sounds good, Thanks! And thanks for your changes.